CODEOWNERS

```diff
@@ -1,2 +1,11 @@
 # the recorder file can have unexpected impact on users' sites
-src/entrypoints/recorder.ts @team-replay
\ No newline at end of file
+src/entrypoints/recorder.ts @team-replay
+
+src/extensions/surveys/ @PostHog/team-surveys
```

Comment:
@raquelmsmith can you add us as contributors to this repo? thanks

Reply:
Done!